virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -282,6 +282,10 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //4.14 leave in per Dan K
 * You cannot run {VirtProductName} on a single-stack IPv6 cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2193267[*BZ#2193267*])
 
+//CNV-38746
+* When you update from {product-title} 4.12 to a newer minor version, VMs that use the `cnv-bridge` Container Network Interface (CNI) fail to live migrate. (link:https://access.redhat.com/solutions/7069807[])
```

Comment:
Something is confusing here:
The bug talks about 4.12->4.14 EUS upgrade, but the doc says 
```
update from {product-title} 4.12 to a newer minor version
```
which implies the problem will also occur when directly upgrading from 4.12 to 4.13.
But according to the [4.13 docs](https://docs.redhat.com/en/documentation/openshift_container_platform/4.13/html/virtualization/virtual-machines#virt-creating-linux-bridge-nad-cli_virt-attaching-vm-multiple-networks), `cnv-bridge` type is still supported on 4.13 NADs.
Although this doc is a 4.14-specific release note, saying `4.12 to a newer minor version` implies 4.13 as well.

Reply:
@yossisegev I had the same question. See @phoracek's [comment](https://github.com/openshift/openshift-docs/pull/91251#discussion_r2019015380) above.